docs/api.rst

```diff
@@ -1140,6 +1141,57 @@ Get current price and custom price for assets
    :statuscode 500: Internal rotki error
    :statuscode 502: An external service used in the query such as cryptocompare/coingecko could not be reached or returned unexpected response.
 
+
+Get current price and custom price for assets
```

Comment:
```suggestion
Get current price and custom price for assets
```

Is this current and custom? Or just always latest price? Or only latest manual? The title is confusin